src/astroFactory.js

```diff
@@ -0,0 +1,81 @@
+import { MeshBuilder } from "@babylonjs/core/Meshes/meshBuilder"
+import { Texture } from "@babylonjs/core/Materials/Textures/texture"
+import { StandardMaterial } from "@babylonjs/core/Materials/standardMaterial"
+import { Scalar } from "@babylonjs/core/Maths/math.scalar"
+import { Animation } from "@babylonjs/core/Animations/animation"
+
+import logger from "./logger";
+
+import distortTexture from "../assets/textures/distortion.png";
+import rockTextureN from "../assets/textures/rockn.png";
+import rockTexture from "../assets/textures/rock.png";
+
+class AstroFactory {
+    static createPlanet(opts, scene) {
+        let planet = MeshBuilder.CreateSphere(opts.name, { diameter: 1 }, scene);
```

Comment:
If you really want to optimize things you can load the `SphereBuilder` instead. But that's nitpicking :-)

Reply:
It's a good demonstration of the need to really get specific in your imports, no worries 👍